src/products/newsStory/formatted-type.ts

```diff
@@ -6,7 +6,7 @@ import { PublishedEntity } from '@/types/formatted/publishedEntity'
 import { MediaImage } from '@/types/formatted/media'
 
 export type NewsStoryTeaser = PublishedEntity & {
-  headingLevel?: ComponentType | keyof JSX.IntrinsicElements
+  headingLevel?: ComponentType | keyof React.JSX.IntrinsicElements
```

Comment:
React 19 no longer uses the global `JSX` namespace, so all this is scoped under the `React` namespace.